### PR TITLE
fixed the compiler issue

### PR DESCRIPTION
**Pull request by brightening-eyes**
by this commit, pyfsuipc can be compiled using every other compiler like vs 2015, vs 2017, etc…'s source for compatibility with the current compiler

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -4,9 +4,6 @@
 
 setup(
     ext_modules = cythonize([
-    Extension("pyfsuipc", ["pyfsuipc.pyx"],
-              include_dirs=['./include'],
-              library_dirs=['./lib'],
-              libraries=["FSUIPC_User"])
+    Extension("pyfsuipc", sources=["pyfsuipc.pyx", "IPCuser.c"], libraries=["advapi32", "user32", "kernel32", "ole32", "oleaut32", "gdi32", "gdiplus", "imm32"])
```
Comment (voneiden) [inline setup.py]: In the spirit of [PEP-8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length), line length should not exceed 99 chars.

**Review by voneiden — CHANGES_REQUESTED** [review]
Thanks for your contribution! Minor change requested in the setup.py regarding line breaks.

If you'd like, I can add you as a maintainer to this repo as well.

**Comment by brightening-eyes** [comment]
hi,
I tried to fix at least the pep-8 error
let me know if there are any other issues.

**Review by voneiden — APPROVED** [review]

**Comment by voneiden** [comment]
Thanks!

**Pull request merged**